src/app/shared/entity/cluster.ts

```diff
@@ -125,6 +125,7 @@ export class AzureCloudSpec {
   tenantID: string;
   vnet: string;
   loadBalancerSKU: string;
+  assignAvailabilitySet: string;
```

Comment:
Shouldn't this one be boolean?

Reply:
I haven't pushed my latest changes as I was still checking out how to solve the issue described above and the pull request is still work in progress. I have it fixed locally.